OpenSim/Examples/ExampleHopperDevice/helperMethods.h

```diff
@@ -184,7 +195,7 @@ inline void simulate(Model& model, SimTK::State& state, bool saveStatesFile)
 
     // Simulate until the user presses ESC (or enters 'q' if visualization has
     // been disabled).
-    while (true) {
+    while (!simulateOnce) {
```

Comment:
> this will result in no simulation, not even once

Hm, perhaps we need to revise [point 3 in CONTRIBUTING.md under "Making a Pull Request (PR)"](https://github.com/opensim-org/opensim-core/blob/master/CONTRIBUTING.md#making-a-pull-request-pr) to indicate that tests must pass *and* the developer must ensure that the code executes as designed?